Blog.API/Blog.Dependencies/Dependencies.cs

```diff
@@ -43,6 +46,11 @@ private static void ConfigureMapster(this IServiceCollection services)
         services.AddSingleton<IMapper>(sp => new ServiceMapper(sp, config));
     }
 
+    private static void ConfigureUnitOfWork(this IServiceCollection services)
+    {
+        services.AddScoped<IUnitOfWork, UnitOfWork>();
```

Comment:
Update DI. Maybe something like this:

```suggestion
services.AddScoped(typeof(IGenericRepository<>), typeof(GenericRepository<>));
services.AddScoped(typeof(IUnitOfWork<>), typeof(UnitOfWork<>));
```